x/stakeibc/keeper/msg_server.go

```diff
@@ -277,6 +285,23 @@ func (k msgServer) RegisterHostZone(goCtx context.Context, msg *types.MsgRegiste
 	return &types.MsgRegisterHostZoneResponse{}, nil
 }
 
+func (ms msgServer) UpdateHostZoneParams(goCtx context.Context, msg *types.MsgUpdateHostZoneParams) (*types.MsgUpdateHostZoneParamsResponse, error) {
```

Comment:
Is this added as a function so governance can call this function? If so, is there any additional wiring needed (e.g. do we need to register anything with the gov module)?

Reply:
yup it's a gov function! the only part that's needed is the `authority` address check

Reply:
Gotcha, perfect then! We might have already done this, but should make sure in Dockernet (or localstride or another env) that Governance can call this function, but other accounts can't

Reply:
yup, already tested in dockernet!